v2.4/getting-started/kubernetes/installation/hosted/kubeadm/index.md

```diff
@@ -4,7 +4,11 @@ redirect_from: latest/getting-started/kubernetes/installation/hosted/kubeadm/ind
 ---
 
 This document outlines how to install Calico, as well as a as single node
-etcd cluster for use by Calico on a Kubernetes cluster created by kubeadm.
+etcd cluster for use by Calico on a Kubernetes cluster created by kubeadm. 
```

Comment:
I created a PR against your branch to show the spaces at the end of the lines.  https://github.com/hickey/calico/pull/1
Feel free just to update your commit with the changes.